analysis/metrics.py

```diff
@@ -55,6 +59,10 @@ class SumMetrics:
       std_cross_partition_error: the standard deviation of the error due to cross-partition contribution bounding.
       std_noise: the noise standard deviation.
       noise_kind: the type of noise used.
+
+  s.t. the following holds:
+  Ex(sum_after_contribution_bounding) = sum + Ex(error)
```

Comment:
Ex -> `Expectation` or `E` (and then to note that E is expectation)
Ex is not much used as a shortcut for expectation